README.md

```diff
@@ -1,2 +1,31 @@
 # hw01
 Repository for homework 01
+
+
+# My name is Kangkana Koli 
+
+
+I'm from **Detroit, MI**
+
+
+I study *Political Science*
+
+Here are some things I enjoy doing!
+
+
+ * Dancing
+ * Traveling
+ * Reading
+ * Listening to Rap/Hip-Hop/R&B
+ 
+ 
+ Here is a link to my favorite **Frank Ocean** song:
+ 
+ 
+ [LINK](https://www.youtube.com/watch?v=r4l9bFqgMaQ)
+ 
+
+### Here is a picture from Garfield Conservatory, one of my favorite places in Chicago!
+
+
+![](IMG-1787.JPG)
```

Comment:
Here in the `[]` you can add the alternative words in case the picture didn't appear.